lib/cob_index/macros/custom.rb

```diff
@@ -537,9 +537,13 @@ def suppress_items
           missing = rec.fields("ITM").select { |field| field["u"] == "MISSING" }
           technical = rec.fields("ITM").select { |field| field["u"] == "TECHNICAL" }
           unwanted_library = rec.fields("HLD").select { |field| field["b"] == "EMPTY" || field["c"] == "UNASSIGNED" }
```

Comment:
Should we remove the definitions that are no longer used?

Reply:
Good catch.  I updated it.